velox/common/memory/Memory.h

```diff
@@ -247,6 +249,12 @@ class MemoryManager final : public IMemoryManager {
   mutable folly::SharedMutex mutex_;
   std::atomic_long totalBytes_{0};
   std::unordered_map<std::string, std::weak_ptr<MemoryPool>> pools_;
+
+  const bool checkUsageLeak_;
```

Comment:
Put const member first?

Reply:
I'm grouping the debug members together to be separated from others. I think this will be clearer

Reply:
Let's keep const first and then group them in const or non-const section? Thanks!